tests/AutoReview/GAE2ETest.php

```diff
@@ -31,7 +31,7 @@ class GAE2ETest extends TestCase
 
     public function test_github_actions_executes_all_the_e2e_tests(): void
     {
-        $expected = array_diff(E2ECollector::getE2ENames(), self::IGNORED_E2E_TESTS);
+        $expected = array_values(array_diff(E2ECollector::getE2ENames(), self::IGNORED_E2E_TESTS));
```

Comment:
Fix this error https://github.com/humbug/php-scoper/actions/runs/10667932194/job/29566542033?pr=1061

assertEqualsCanonicalizing seems to still care about key differences, it just doesn't care about the key order.

Reply:
Extracted it in #1070